composite-actions/nodejs-script/print-topology/action.yml

```diff
@@ -15,7 +15,8 @@ inputs:
     default: 'README.md'
   applicationIds:
     description: 'List of application ids provided as a JSON array string. For example: ["application-1", "application-2"]'
```

Comment:
Let's make it a little bit more descriptive:
List of application ids (processor with multiple processing-functions will have many) provided as a JSON array string. For example: ["application-1", "application-2"]

Reply:
Ok, updated the description of this input parameter